packages/debug/src/browser/editor/debug-inline-value-decorator.ts

```diff
@@ -194,7 +194,7 @@ export class DebugInlineValueDecorator implements FrontendApplicationContributio
                                     }
                                     if (expr) {
                                         const expression = new ExpressionItem(expr, () => stackFrame.thread.session);
-                                        await expression.evaluate('watch');
+                                        await expression.evaluate('watch', false);
```

Comment:
Personally, i find this code confusing. I know what it does, but it is strange that this is the only place where we need to set the resolveLazy to false. I believe the code would actually be more intuitive if the default value for resolveLazy in the evaluate method would be false and we explicitly set it to true when needed. It is "more code" but more intuitive imho as the default value of the option is also off/false. WDYT?